src/WebJobs.Extensions.CosmosDB/Services/CosmosDBService.cs

```diff
@@ -15,15 +16,27 @@ internal sealed class CosmosDBService : ICosmosDBService, IDisposable
         private bool _isDisposed;
         private DocumentClient _client;
 
-        public CosmosDBService(string connectionString, ConnectionPolicy connectionPolicy)
+        public CosmosDBService(string connectionString, ConnectionPolicy connectionPolicy, bool useDefaultJsonSerialization)
```

Comment:
@ealsur This looks like a breaking change on a public type.. are you sure its OK?

Reply:
nevermind, I misread, this is an internal type